src/server/services/wmo/wmo.test.ts

```diff
@@ -169,7 +170,7 @@ describe('Transform api items', () => {
         },
       ];
 
-      const result = getDisclaimer(aanvraag, aanvragen);
+      const result = getHulpmiddelenDisclaimer(aanvraag, aanvragen);
       expect(result).toMatch(
         'Dit hulpmiddel staat per ongeluk ook bij "Eerdere en afgewezen voorzieningen". Daar vindt u het originele besluit met de juiste datums.'
       );
```

Comment:
Geen regex of substring in toMatch?
Dan denk ik beter:
```suggestion
      expect(result).toBe(
        'Dit hulpmiddel staat per ongeluk ook bij "Eerdere en afgewezen voorzieningen". Daar vindt u het originele besluit met de juiste datums.'
      );
```
```